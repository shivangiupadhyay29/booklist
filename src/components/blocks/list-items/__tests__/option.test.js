import React from 'react';
import Listing from '../index.js';
import { shallow, mount } from 'enzyme';
import Option from '../option';

describe('list-options', () => {
  it('list -item for options component unit testing', () => {
    const wrapper = shallow(
      <Option classText={'label-size'} content={'testing content'} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
