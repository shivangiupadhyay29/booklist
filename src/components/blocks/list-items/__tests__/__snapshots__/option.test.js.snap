// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list-options list -item for options component unit testing 1`] = `ShallowWrapper {}`;
